### PR TITLE
fmt(cstyle): cstyle fixes in istgt_ser_table.c

### PR DESCRIPTION
**Pull request by sushma-priyadharssini**

### CODE REVIEW

File: src/istgt_ser_table.c [hunk 1]
```diff
@@ -93,24 +95,44 @@
...
-/*REL     */{   bK, bK, bK,  bK,  bK,  bK,  bK,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  bK,    pS},
-/*LOG_SNS */{   pS, pS, pS,  pS,  pS,  bK,  bK,  pS,  pS,  bK,  bK, pS,  bK, bK, bK,  bK,  bK,    pS},
-/*FORMAT  */{   pS, bK, bK,  bK,  bK,  bK,  pS,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  bK,    pS},
-/*START   */{   pS, bK, bK,  bK,  bK,  bK,  bK,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  bK,    pS},
-/*PRES_IN */{   bK, bK, bK,  pS,  bK,  bK,  bK,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  bK,    pS},
-/*PRES_OUT*/{   bK, bK, bK,  bK,  bK,  bK,  bK,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  bK,    pS},
-/*MAIN_IN */{   pS, bK, bK,  bK,  bK,  bK,  bK,  pS,  bK,  bK,  bK, bK,  bK, bK, bK,  bK,  pS,    pS},
-/*INVLD   */{   pS, pS, pS,  pS,  pS,  pS,  pS,  pS,  pS,  pS,  pS, pS,  pS, pS, pS,  pS,  pS,    pS}
+/*
+ * >IDX_ :: 2nd:TUR RD  WRT UNMP MDSN MDSL RQSN INQ  RDCP RES  REL LSNS
```
Comment (vishnuitta) [inline src/istgt_ser_table.c]: For easy reading of the table, can you please change these lines as
```
 * >IDX_ :: 2nd:
 * TU RD WRT UNMP MSN MDS RQS INQ RDC RES REL LSN FMT STR PR PROT MAI INV
 * R                  L   N       P           S           IN      NIN LD
```

**Review by vishnuitta — CHANGES_REQUESTED** [review]
changes required as per one comment from me. Thanks.

File: src/istgt_ser_table.c (hunk 1 above)
Comment (sushma-priyadharssini) [reply src/istgt_ser_table.c]: Updated the comment as suggested.

**Review by vishnuitta — APPROVED** [review]
changes are good

**Pull request merged**